src/Raven.Server/Documents/Subscriptions/SubscriptionConnectionsState.cs

```diff
@@ -726,7 +726,7 @@ private SubscriptionConnectionsState(DocumentsStorage storage, SubscriptionState
         private long _lastNoopAckTicks;
         private Task _lastNoopAckTask = Task.CompletedTask;
 
-        public Task SendNoopAck()
+        public Task SendNoopAck(bool force = false)
```

Comment:
I do not see a place with 'force = true' ?

Reply:
right should be here:

https://github.com/garayx/ravendb/blob/f1e6b4fac577107c158d97405b9ad72862d9e542/src/Raven.Server/Documents/TcpHandlers/SubscriptionConnection.cs#L241

I forgot to add the file to the commit :)